docs/reference/training/experiment-config-reference.rst

```diff
@@ -318,6 +318,22 @@ Example configuration:
 These settings may also be specified at the cluster or resource pool level through task container
 defaults.
 
+.. _log-retention-days:
+
+``log_retention_days``
+======================
+
+Optional. Overrides the number of days to retain logs for a trial. If set to `-1`, logs will be
+retained indefinitely. If set to `0`, logs will be deleted during the next cleanup.
+
+Example configuration:
+
+.. code:: yaml
+
+   log_retention_days: 90
```

Comment:
It introduces a bit of complexity to have differing syntax for whether you're setting the default retention policy in the master or setting the retention policy for this one particular experiment. I think either syntax works for me, I just don't want to have to remember both and keep them straight.

Checking in, is this the desired choice? For comparison, `log_policies` has both the same syntax and same name in both master config and experiment config.

Reply:
The names are somewhat unfortunate but due to collisions with log/logging naming in the configs this seems like the best compromise. The server can also take a schedule and default number of days for retention, while the experiment can only configure the number of days and not the schedule.